src/thunderbots/software/test/navigator/path_planner/astar.cpp

```diff
@@ -0,0 +1,71 @@
+//
+// Created by jordan on 2/26/19.
+//
```

Comment:
Could we change this to a proper start-of-file comment?

Reply:
added